src/main/java/uk/gov/hmcts/reform/sscs/personalisation/SyaAppealCreatedAndReceivedPersonalisation.java

```diff
@@ -157,7 +157,7 @@ private String buildHearingDetails(HearingOptions hearingOptions) {
                 .append("Attending the hearing: ")
                 .append(hearingOptions.getWantsToAttend().toLowerCase());
 
-        if (hearingOptions.getWantsToAttend().toLowerCase().equals("yes") && hearingOptions.getExcludeDates() != null && hearingOptions.getExcludeDates().size() > 0) {
+        if (hearingOptions.getWantsToAttend().equalsIgnoreCase("yes") && hearingOptions.getExcludeDates() != null && !CollectionUtils.isEmpty(hearingOptions.getExcludeDates())) {
```

Comment:
maybe nicer to use StringUtils.equalsIgnoreCase as it checks for nulls, from what I remember, unless Java8 has done a better job with null checks